app/components/mandatarissen/update-state.js

```diff
@@ -150,11 +150,8 @@ export default class MandatarissenUpdateState extends Component {
   }
 
   get isInputDateTheSameAsMandatarisStart() {
-    const startDate = new Date(this.args.mandataris.start);
-    startDate.setHours(0, 0, 0, 0);
+    const startDate = new Date();
     const inputDate = new Date(this.date);
-    inputDate.setHours(0, 0, 0, 0);
-
     return startDate.getTime() === inputDate.getTime();
```

Comment:
isn't it possible that this fails if we're right on the boundary of the millisecond? I agree that the chance is small... Maybe we should do something like the code before tries to do, maybe using momentjs?